var/spack/repos/builtin/packages/py-pytest-allclose/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPytestAllclose(PythonPackage):
+    """Pytest fixture extending Numpy's allclose function."""
+
+    pypi = "pytest-allclose/pytest-allclose-1.0.0.tar.gz"
+
+    maintainers("paugier")
+
+    license("MIT", checked_by="paugier")
+
+    version("1.0.0", sha256="b2f0c521fa652281400d4a105c84454db3c50b993bcfee9861380be69cc6b041")
+
+    extends("python@3.9:", type=("build", "run"))
```

Comment:
Sorry, I overlooked this earlier.
```suggestion
    depends_on("python@3.9:", type=("build", "run"))
```
Within PythonPackages, just a depends_on for a specific minimum python version. The extends is for non-PythonPackages that have optional python installation capabilities (think CMakePackage with optional python bindings). The docs add more details at https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#extends-vs-depends-on, but essentially it's sufficient to know that the PythonPackage base class already has the extends directive so it's not needed here.

Reply:
Thanks for the explanation!